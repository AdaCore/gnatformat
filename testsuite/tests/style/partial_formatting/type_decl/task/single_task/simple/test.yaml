description: SingleTaskDecl formatting for line selection.
driver: partial_gnatformat
args:
  [
    "test.ads",
    "-SL 138",
    "-EL 139",
  ]
status_code: 1