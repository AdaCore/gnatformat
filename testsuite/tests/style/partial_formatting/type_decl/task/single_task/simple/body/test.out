<CompilationUnit test.adb:26:1-77:31>
test.adb(<TaskBody ["T"] test.adb:63:4-70:10>) - 63:1-70:10
^
   task body T is
   begin
      Put_Line ("Task T will delay for 4 seconds...");
      delay 4.0;
      Put_Line ("Task T will set Obj...");
      Obj.Set (5);
      Put_Line ("Task T has just set Obj...");
   end T;$

